src/igvfd/tests/test_audit_in_vitro_system.py

```diff
@@ -18,7 +23,7 @@ def test_audit_targeted_sample_term(
     )
 
 
-def test_audit_targeted_sample_term(
+def test_audit_cell_fate_change_treatments(
```

Comment:
Coverage was decreasing in part because this test had a duplicated name to the above test, causing the latter to not be tested by pytest/coveralls. Also the test above was broken so I fixed it.